portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -17,11 +22,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String company = request.getParameter("company");
     String email = request.getParameter("email");
     String message = request.getParameter("text-input");
+    
+    // Store contact info in datastore
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    KeyFactory keyFactory = datastore.newKeyFactory().setKind("Task");
+    FullEntity taskEntity =
+        Entity.newBuilder(keyFactory.newKey())
+            .set("name", name)
+            .set("company", company)
+            .set("email", email)
```

Comment:
nit: Something to consider when you have the same String literal repeated multiple times in different places is to make it a constant, which can make later refactoring easier and help catch bugs earlier. For example, a typo in the String literal will be hard to track down but a typo in the constant name will be a compiler error.